javascripts/discourse/api-initializers/discourse-topic-list-author.gjs

```diff
@@ -0,0 +1,76 @@
+import { eq } from "truth-helpers";
+import bodyClass from "discourse/helpers/body-class";
+import { withPluginApi } from "discourse/lib/plugin-api";
+import AuthorColumnContent from "../components/author-column-content";
+import AuthorColumnHeader from "../components/author-column-header";
+import CustomCategoriesLatestAvatar from "../components/custom-categories-latest-avatar";
+
+export default {
+  name: "discourse-topic-list-cards",
+  initialize() {
+    withPluginApi("1.39.0", (api) => {
```

Comment:
This is in the `api-initializers` directory, but not actually implemented using `apiInitializer()`

We should either move it to the normal `initializers` directory, or switch it to use `apiInitializer`

Reply:
Done